_includes/docs/v2.23/setup-script-credentials-gke.md

```diff
@@ -2,6 +2,20 @@
 . ./helpers/setup.sh -p PROJECT_ID -o ORG_ID -k
 ```
 
-This will create a service account called `cloud-foundation-forseti-<suffix>`,
-assign it the proper roles, and download the service account credentials to
-`${PWD}/credentials.json`.
+This will create a service account called `cloud-foundation-forseti-<suffix>`
+Grant the newly created service account the following roles:
+- roles/container.admin
+- roles/container.clusterAdmin
+- roles/container.developer
+- roles/iam.serviceAccountKey.Admin
+- roles/resourcemanager.projectIamAdmin
+- roles/compute.networkAdmin
+
+Ensure the following APIs are enabled on the Forseti project:
+- container.googleapis.com
+- compute.googleapis.com
+
+Set and export the `GOOGLE_APPLICATION_CREDENTIALS` variable to the newly
+created credential.json as shown below.
```

Comment:
credential.json --> credentials.json

Also same comment as above

Reply:
Removed this sentence as this is covered in setup.sh script.